package/gluon-mesh-batman-adv/src/respondd.c

```diff
@@ -444,74 +444,6 @@ static struct json_object * get_traffic(void) {
 	return ret;
 }
 
-static void count_iface_stations(size_t *wifi24, size_t *wifi5, const char *ifname) {
```

Comment:
`count_iface_stations` has already been moved to respondd-neighbours.c in 0f1fa243f